Cargo.toml

```diff
@@ -186,19 +186,19 @@ opt-level = "s"
 debug = true
 overflow-checks = false
 
-[profile.dev]
-split-debuginfo = "unpacked"
-overflow-checks = false
-# wait until https://github.com/rust-lang/rust/issues/100142 fixed
-incremental = false
-
-[profile.dev.package]
-addr2line = { opt-level = 3 }
-adler = { opt-level = 3 }
-gimli = { opt-level = 3 }
-miniz_oxide = { opt-level = 3 }
-object = { opt-level = 3 }
-rustc-demangle = { opt-level = 3 }
+# [profile.dev]
```

Comment:
what's this change for ?

Reply:
It's a mistake and it's for local debugging. Recover it soon,